lib/identity_doc_auth/acuant/request.rb

```diff
@@ -1,5 +1,16 @@
 module IdentityDocAuth
   module Acuant
+    # https://documentation.help/AssureID-Connect/Error%20Codes.html
+    # 438 and 439 are frequent errors that we do not want to be notified of
+    IGNORED_ERROR_CODES = [438, 439]
```

Comment:
let's make it a set?

```suggestion
    IGNORED_ERROR_CODES = Set[438, 439]
```